modules/azure/src/main/java/org/apache/ignite/spi/discovery/tcp/ipfinder/azure/TcpDiscoveryAzureBlobStoreIpFinder.java

```diff
@@ -358,6 +368,26 @@ private void removeContainer(String containerName) {
         }
     }
 
+    /** Handles stripping the interface name for IpV6 addresses */
+    private static void handleInterfaceStripping(Collection<InetSocketAddress> addrs) {
```

Comment:
We do not need to remove IPv6 addresses from finder, we only need to remove "%.*$" from the addresses before saving them to the storage.

Reply:
Then they should be saved all right and also read all right on remote side.